SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -62,7 +62,7 @@ class ApplicationPresenter : ApplicationContract.Presenter() {
 
         // Get the content of an URL.
         val response: HttpResponse =
-            client.get<HttpResponse>("https://mobile-api-dev.lner.co.uk/v1/fares?originStation=$originCode&destinationStation=$destinationCode&outboundDateTime=2020-07-15T12%3A16%3A27.371%2B00%3A00&inboundDateTime=2020-03-06T12%3A16%3A27.371%2B00%3A00&numberOfChildren=1&numberOfAdults=0&doSplitTicketing=false")
+            client.get<HttpResponse>("https://mobile-api-dev.lner.co.uk/v1/fares?originStation=$originCode&destinationStation=$destinationCode&outboundDateTime=2020-07-16T12%3A16%3A27.371%2B00%3A00&inboundDateTime=2020-03-06T12%3A16%3A27.371%2B00%3A00&numberOfChildren=1&numberOfAdults=0&doSplitTicketing=false")
```

Comment:
We might want to (on a separate PR) make it so that it fetches today's journeys, rather than hard-code a day.